tests/test_print rich.py

```diff
@@ -73,8 +73,8 @@ def test_print_alphabetical(capsys: CaptureFixture[str], option_simple: Options,
     assert output[0].startswith("🐍 Distributions within")
     assert output[1] == sys.executable or output[0].endswith(sys.executable)
     assert output[-4:] == [
-        "├── a 1 remote 2 a month ago",
```

Comment:
Wonder why this change?

Reply:
The original issue was that you were using `naturaltime` whose type signature accepts `datetime|int`, which mypy was flagging. Looking into the `humanize` library, I realised that we should move to use `naturaldelta` that has the right signature (takes a `timedelta`). However, that function doesn't include `ago` in the output.

Reply:
TBH looking at the code of `humanize` the typing is inconsistent, you may consider to contribute a patch there.

https://github.com/python-humanize/humanize/blob/main/src/humanize/time.py#L208 <<< `naturaltime` signature doesn't take a timedelta - however, internally it accepts it https://github.com/python-humanize/humanize/blob/main/src/humanize/time.py#L238